test/utils/date_time_extension_test.dart

```diff
@@ -0,0 +1,337 @@
+import 'package:fluffychat/utils/date_time_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('[localizedTimeShort]', () {
+    testWidgets(
+        'should return time of day with format hh:mm when date is the same day',
```

Comment:
IMO: I recommend for you about clean way to write descriptions.  You can refer to `test/domain/contacts/contacts_manager_test.dart`